phpBB/language/en/install.php

```diff
@@ -355,17 +355,40 @@
 
 		<br />
 
+		<h1>How to update your installation with the Full Package</h1>
+
+		<p>The recommended way of updating your installation is using the full package. If core phpBB files have been modified in your installation you may wish to use the automatic update package in order to not lose these changes. You are also able to update your installation using the other methods listed within the INSTALL.html document. The steps for updating phpBB3 using the full package are:</p>
+
+		<ol style="margin-left: 20px; font-size: 1.1em;">
+			<li><strong class="error">Backup all board files and the database.</strong></li>
+			<li>Go to the <a href="https://www.phpbb.com/downloads/" title="https://www.phpbb.com/downloads/">phpBB.com downloads page</a> and download the latest "Full Package" archive.</li>
+			<li>Unpack the archive.</li>
+			<li>Remove (delete) the <code class="inline">config.php</code> file, and the <code class="inline">/images</code>, <code class="inline">/store</code> and <code class="inline">/files</code> folders <em>from the package</em> (not your site).</li>
```

Comment:
why do we recommend to remove the images/, too? Adding new images e. g. smilies won't be recognized by updaters.

Reply:
It's the same text as here so I'd actually keep it like that: https://www.phpbb.com/support/docs/en/3.2/ug/upgradeguide/upgrade31/

We might employ emojis more often so I'd say it's ok like that.

Reply:
I'm. fine with it.